spec/features/editing_attachments/upload_attachment_spec.rb

```diff
@@ -29,5 +30,9 @@ def and_i_upload_an_attachment
     find('form input[type="file"]').set(Rails.root.join(file_fixture(@attachment_filename)))
     fill_in "title", with: "A title"
     click_on "Upload"
+
+  def then_i_can_see_the_attachment_markdown_on_the_page
```

Comment:
Maybe this should be renamed to - `then_i_can_see_the_attachment_markdown`?

Reply:
Updated